qiskit/chemistry/drivers/fcidumpd/dumper.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" FCIDump dumper """
```

Comment:
According to the doc guidelines there should be no leading or trailing whitespaces and full sentences should be used (or at least pseudo-sentences with a period at the end...)

```suggestion
"""FCIDump dumper."""
```